src/error.rs

```diff
@@ -15,16 +15,12 @@ use core::fmt;
 ///            Err(Error::InvalidValue));
 /// # }
 /// ```
-#[derive(Clone, Copy, Debug, PartialEq, Eq, Hash)]
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
```

Comment:
Why?

Reply:
I guess I can postpone this change to a separate PR (yay or nay?), but it would be interesting for the error to retain the receiving vector if it could not be transmuted (so as to make errors fully recoverable; `Vec` is passed by value, after all). With this in mind, this error type is likely to be unable to implement `Copy` in the future.

Reply:
Valid, keep 'er in